R/data-initial-placeholder.R

```diff
@@ -0,0 +1,11 @@
+
+#' Initial Placeholder Data
+#'
+#' The initial data.frame is generated as a placeholder until a more streamlined
+#' process is implemented. It contains the initial run where all of CRAN was
+#' scored using riskmetric v0.2.1 on 2023-06-21. It contains 19,715 observations
+#' (one row per package) and 24 variables, 18 of which are riskmetric
+#' assessments.
+#'
```

Comment:
Recommend also including a listing of variable names (this should be part of the `@format` tag contents). You can see an example of this in the various base data packages (eg `?survival::lung`)

```suggestion
#' \describe{
#'   \item{varname}{description of varname}
#'   \item{varname2{description of varname2}
#'   ...
#' }
```

Reply:
Complete! 
![image](https://github.com/pharmaR/riskscore/assets/17878953/a5c4333b-ca9e-4da1-bbbc-04e67687f6b3)